File: coding-flink-10/src/main/java/org/ace/flink/java/TestSource.java
package org.ace.flink.java;

import org.ace.flink.source.AutoGenerateSource;
import org.apache.flink.streaming.api.datastream.DataStream;
import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;

public class TestSource {
    public static void main(String[] args) throws Exception {
        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
        DataStream lines = env.addSource(new AutoGenerateSource());

        lines.print("source");
        env.execute("testSource");
    }
}

File: coding-flink-11/src/main/scala/org/ace/flink/test/TestSource.scala
package org.ace.flink.test

import org.ace.flink.source.AutoGenerateSource
import org.apache.flink.streaming.api.scala._

object TestSource {
  def main(args: Array[String]): Unit = {
    val env = StreamExecutionEnvironment.getExecutionEnvironment
    val lines = env.addSource(new AutoGenerateSource())

    lines.print("source")
    env.execute("testSource")
  }


}
